packages/terra-core-docs/src/terra-dev-site/doc/button/example/ButtonAnchor.jsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
-import Button from 'terra-button/lib/Button';
+import AnchorButton from 'terra-button/lib/variants/_AnchorButton';
```

Comment:
same here 
`import { AnchorButton } from 'terra-button';`

Reply:
updated in https://github.com/cerner/terra-core/pull/3655/commits/81401caf927c1b11a30aa72ed0740f8e7e2c6df6